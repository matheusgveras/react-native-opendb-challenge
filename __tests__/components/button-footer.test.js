import 'react-native';
import React from 'react';
import ButtonFooter  from '../../src/components/button-footer';
import renderer from 'react-test-renderer';
describe('testing button footer component', () => {
    
    it('renders buttom', () => {
        const view =  <ButtonFooter onPress={() => {}} label={'Testing'}  />
        const tree = renderer.create(view).toJSON();
        expect(tree).toMatchSnapshot();
    });

});