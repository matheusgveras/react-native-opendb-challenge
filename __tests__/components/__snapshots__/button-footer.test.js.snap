// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing button footer component renders buttom 1`] = `
<View>
  <View
    accessible={true}
    clickable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#00306B",
        "borderRadius": 0,
        "borderTopColor": "#00306B",
        "borderTopWidth": 1,
        "height": 75,
        "justifyContent": "center",
        "opacity": 1,
        "paddingHorizontal": 15,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#00306B",
          "flexDirection": "row",
          "marginHorizontal": 10,
        }
      }
    >
      <Text
        bold={true}
        style={
          Object {
            "color": "#fff",
            "flex": 1,
            "fontWeight": "bold",
            "letterSpacing": 2,
          }
        }
        upper={true}
      >
        Testing
      </Text>
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": undefined,
              "fontSize": 12,
            },
            Object {
              "color": "#B5B5B5",
            },
            Object {
              "fontFamily": "Ionicons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
  </View>
</View>
`;
